tuf/client/client.go

```diff
@@ -421,8 +419,9 @@ func (c *Client) downloadTargets(role string) error {
 	return nil
 }
 
-func (c *Client) downloadSigned(role string, size int64, expectedSha256 []byte) ([]byte, *data.Signed, error) {
-	raw, err := c.remote.GetMeta(role, size)
+func (c *Client) downloadSigned(role string, size int64, expectedSha256 []byte, consistent bool) ([]byte, *data.Signed, error) {
```

Comment:
I would rather not change the API and just infer whether we have a sha256 or not (if you don't want consistent download, pass `nil`)